namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-query-development/resources/rds.tf

```diff
@@ -3,6 +3,17 @@
 # Application RDS (PostgreSQL)
 #################################################################################
 
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
```

Comment:
You will not need these once you have added variables.tf. Please update the branch with master

Reply:
Yes, thank you, I realised that belatedly :) 

Wanted to see it pass then I was going to merge master